src/main/java/org/lookout_studios/meals_management_system/meals_management_system/mysqlConnection.java

```diff
@@ -0,0 +1,30 @@
+package org.lookout_studios.meals_management_system.meals_management_system;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class MysqlConnection {
+    public void sqlConnection() {
+        String url = "jdbc:mysql://localhost:3306/test"; /* Change `test` to desired database.*/
+        String username = "root";
+        String password = "";
+        try {
+            Class.forName ("com.mysql.cj.jdbc.Driver");
+            Connection connection = DriverManager.getConnection (url, username, password);
+            Statement statement = connection.createStatement ();
+            ResultSet resultSet = statement.executeQuery ("select * from students"); /* Change `students` to desired table. */
+            /* For tests only. */
+            while (resultSet.next ()) {
+                System.out.println(
+                    resultSet.getInt (1)+" "+resultSet.getString (2)+" "+resultSet.getInt (3));
+            }
+            connection.close ();
+        }
+        catch (Exception exception) {
+            System.out.println ("An error has occured.");
```

Comment:
Avoid using magic values. Use a static variable at the beginning of the file or remove this line altogether.

Reply:
It is impossible not to use catch function in this form. Even [official documentation](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-usagenotes-connect-drivermanager.html) shows this kind of practice.

Reply:
SQL demands usage of this practice as the connection establishment enforces that.

Reply:
My comment does not refer to any of the above but to the magic value you used. Refrain from using them, if possible.